coreos-4/policies/RA-Risk_Assessment/component.yaml

```diff
@@ -117,34 +117,33 @@
 - control_key: RA-5
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: partial
+  implementation_status: not applicable
   narrative:
     - key: a
       text: |
-        This control reflects organizational procedures/policies, and is not
-        applicable to the configuration of CoreOS.
+        Organizational procedures to initially scan for vulnerabilities and to perform a scan
+        when new vulnerabilities are identified and reported are outside the scope of RHCOS
+        configuration.
     - key: b
       text: |
-        A control response is planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-352
+        Organizational procedures to facilitate interoperability and vulnerability management
+        process automation are outside the scope of RHCOS configuration.
```

Comment:
NACK. For RA-5(b) a response that outline OpenShift's embedded vulnerability scanning capabilities would be appropriate.